server/client.go

```diff
@@ -634,7 +636,8 @@ func (c *client) writeLoop() {
 	// buffered outbound structure for efficient writev to the underlying socket.
 	for {
 		c.mu.Lock()
-		if waitOk && (c.out.pb == 0 || c.out.fsp > 0) && len(c.out.nb) == 0 && !c.flags.isSet(clearConnection) {
+		owtf := c.out.fsp > 0 && c.out.pb < maxBufSize && c.out.fsp < 10
```

Comment:
`c.out.fsp < 10` should we create a constant for the `10` here too? `maxFlushSignalsPending` maybe?

Reply:
fixed